Pachli Current: App crashes on startup.

**Describe the bug**
Since the last update, the app crashes when it is opened after a longer time.
Clearing the app cache does not help.
Clearing the app data helps, brings back the initial login and the app then loads and behaves as expected. When opened after a longer time, the app crashes immediately and does not recover until the app data is deleted.

**Versions**
2.0.0+e749b362

Android 14
Pixel 7a

**Additional context**
i have a feeling that it might somehow be caused by a larger backlog of posts in the feed after reopening and that's why it works fine after the first login/setup, but fails after a longer time without catching up on posts.
However, this does not happen with the stable version. This loads without any problems, no matter how long it was in the background.


**Affirmation**
I have checked other issues for this project (open and closed) and I cannot find one that matches the problem I am reporting.

This data will be sent to the developers.

Please check to make sure it does not include anything you don't want to share, and please describe what you were doing when the crash happened.

----
Your description here:

----

REPORT_ID=9ae6cded-f46f-4b21-bd8e-91175e9aa4be
APP_VERSION_CODE=3005
APP_VERSION_NAME=2.0.0+29f9273c
PACKAGE_NAME=app.pachli.current
FILE_PATH=/data/user/0/app.pachli.current/files
PHONE_MODEL=Pixel 7a
BRAND=google
PRODUCT=lynx
ANDROID_VERSION=14
BUILD=BOARD=lynx
  BOOTLOADER=lynx-14.0-10825040
  BRAND=google
  CPU_ABI=arm64-v8a
  CPU_ABI2=
  DEVICE=lynx
  DISPLAY=UQ1A.231205.015
  FINGERPRINT=google/lynx/lynx:14/UQ1A.231205.015/11084887:user/release-keys
  HARDWARE=lynx
  HOST=abfarm-release-rbe-32-2004-00044
  ID=UQ1A.231205.015
  IS_DEBUGGABLE=false
  IS_EMULATOR=false
  MANUFACTURER=Google
  MODEL=Pixel 7a
  ODM_SKU=GHL1X
  PERMISSIONS_REVIEW_REQUIRED=true
  PRODUCT=lynx
  RADIO=unknown
  SKU=GHL1X
  SOC_MANUFACTURER=Google
  SOC_MODEL=GS201
  SUPPORTED_32_BIT_ABIS=[]
  SUPPORTED_64_BIT_ABIS=["arm64-v8a"]
  SUPPORTED_ABIS=["arm64-v8a"]
  TAGS=release-keys
  TIME=1699657908000
  TYPE=user
  UNKNOWN=unknown
  USER=android-build
  VERSION.ACTIVE_CODENAMES=[]
  VERSION.BASE_OS=
  VERSION.CODENAME=REL
  VERSION.INCREMENTAL=11084887
  VERSION.KNOWN_CODENAMES={HoneycombMr1, HoneycombMr2, Lollipop, Kitkat, Tiramisu, Gingerbread, Cupcake, IceCreamSandwichMr1, JellyBean, IceCreamSandwich, LollipopMr1, M, N, O, P, Q, R, S, Sv2, Base, NMr1, OMr1, JellyBeanMr1, JellyBeanMr2, Donut, Froyo, GingerbreadMr1, EclairMr1, UpsideDownCake, Honeycomb, Eclair01, KitkatWatch, Base11, Eclair}
  VERSION.MEDIA_PERFORMANCE_CLASS=33
  VERSION.PREVIEW_SDK_FINGERPRINT=REL
  VERSION.PREVIEW_SDK_INT=0
  VERSION.RELEASE=14
  VERSION.RELEASE_OR_CODENAME=14
  VERSION.RELEASE_OR_PREVIEW_DISPLAY=14
  VERSION.SDK=34
  VERSION.SDK_INT=34
  VERSION.SECURITY_PATCH=2023-12-05
TOTAL_MEM_SIZE=118015217664
AVAILABLE_MEM_SIZE=25799405568
BUILD_CONFIG=b=false
  d=false
  f=false
  h=false
  i=0
CUSTOM_DATA=
IS_SILENT=false
STACK_TRACE=java.lang.RuntimeException: Unable to start activity ComponentInfo{app.pachli.current/app.pachli.SplashActivity}: java.lang.IllegalArgumentException: No enum constant z6.n.TRENDINGTAGS
at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3802)
at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3942)
at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:103)
at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:139)
at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:96)
at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2448)
at android.os.Handler.dispatchMessage(Handler.java:106)
at android.os.Looper.loopOnce(Looper.java:205)
at android.os.Looper.loop(Looper.java:294)
at android.app.ActivityThread.main(ActivityThread.java:8194)
at java.lang.reflect.Method.invoke(Native Method)
at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:552)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:971)
Caused by: java.lang.IllegalArgumentException: No enum constant z6.n.TRENDINGTAGS
at java.lang.Enum.valueOf(Enum.java:302)
at z6.n.valueOf(SourceFile:1)
at w6.j.a(SourceFile:105)
at u6.c.<init>(SourceFile:867)
at k5.f0.get(SourceFile:716)
at fd.a.get(SourceFile:14)
at f.n.a(SourceFile:149)
at androidx.activity.m.onCreate(SourceFile:30)
at androidx.fragment.app.c0.onCreate(SourceFile:1)
at app.pachli.SplashActivity.onCreate(SourceFile:21)
at android.app.Activity.performCreate(Activity.java:8621)
at android.app.Activity.performCreate(Activity.java:8599)
at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1456)
at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3784)
... 12 more

INITIAL_CONFIGURATION=colorMode=10
  densityDpi=356
  fontScale=1
  fontWeightAdjustment=0
  hardKeyboardHidden=HARDKEYBOARDHIDDEN_YES
  keyboard=KEYBOARD_NOKEYS
  keyboardHidden=KEYBOARDHIDDEN_NO
  locale=en_DE
  mcc=262
  mnc=1
  navigation=NAVIGATION_NONAV
  navigationHidden=NAVIGATIONHIDDEN_YES
  orientation=ORIENTATION_PORTRAIT
  screenHeightDp=1002
  screenLayout=SCREENLAYOUT_SIZE_LARGE+SCREENLAYOUT_LONG_YES+SCREENLAYOUT_LAYOUTDIR_LTR+SCREENLAYOUT_ROUND_NO
  screenWidthDp=485
  seq=430
  smallestScreenWidthDp=485
  touchscreen=TOUCHSCREEN_FINGER
  uiMode=UI_MODE_TYPE_NORMAL+UI_MODE_NIGHT_YES
  userSetLocale=true
CRASH_CONFIGURATION=colorMode=10
  densityDpi=356
  fontScale=1
  fontWeightAdjustment=0
  hardKeyboardHidden=HARDKEYBOARDHIDDEN_YES
  keyboard=KEYBOARD_NOKEYS
  keyboardHidden=KEYBOARDHIDDEN_NO
  locale=en_DE
  mcc=262
  mnc=1
  navigation=NAVIGATION_NONAV
  navigationHidden=NAVIGATIONHIDDEN_YES
  orientation=ORIENTATION_PORTRAIT
  screenHeightDp=1002
  screenLayout=SCREENLAYOUT_SIZE_LARGE+SCREENLAYOUT_LONG_YES+SCREENLAYOUT_LAYOUTDIR_LTR+SCREENLAYOUT_ROUND_NO
  screenWidthDp=485
  seq=430
  smallestScreenWidthDp=485
  touchscreen=TOUCHSCREEN_FINGER
  uiMode=UI_MODE_TYPE_NORMAL+UI_MODE_NIGHT_YES
  userSetLocale=true
DISPLAY=0.currentSizeRange.smallest=[1080,965]
  0.currentSizeRange.largest=[2282,2229]
  0.flags=FLAG_SUPPORTS_PROTECTED_BUFFERS+FLAG_SECURE
  0.metrics.density=2.2250001430511475
  0.metrics.densityDpi=356
  0.metrics.scaledDensity=x2.2250001
  0.metrics.widthPixels=1080
  0.metrics.heightPixels=2229
  0.metrics.xdpi=391.885009765625
  0.metrics.ydpi=409.12701416015625
  0.realMetrics.density=2.2250001430511475
  0.realMetrics.densityDpi=356
  0.realMetrics.scaledDensity=x2.2250001
  0.realMetrics.widthPixels=1080
  0.realMetrics.heightPixels=2400
  0.realMetrics.xdpi=391.885009765625
  0.realMetrics.ydpi=409.12701416015625
  0.name=Integrierter Bildschirm
  0.realSize=[1080,2400]
  0.rectSize=[0,0,1080,2229]
  0.size=[1080,2229]
  0.rotation=ROTATION_0
  0.isValid=true
  0.orientation=0
  0.refreshRate=60
  0.height=2229
  0.width=1080
  0.pixelFormat=1
USER_COMMENT=
USER_EMAIL=
USER_APP_START_DATE=2023-12-14T11:03:32.539+01:00
USER_CRASH_DATE=2023-12-14T11:03:32.879+01:00
DUMPSYS_MEMINFO=
LOGCAT=12-14 11:03:32.477 D/CompatibilityChangeReporter(  435): Compat change id reported: 171979766; UID 10406; state: ENABLED
12-14 11:03:32.477 D/CompatibilityChangeReporter(  435): Compat change id reported: 242716250; UID 10406; state: ENABLED
12-14 11:03:32.484 W/WindowOnBackDispatcher(28611): sendCancelIfRunning: isInProgress=falsecallback=android.view.ViewRootImpl$$ExternalSyntheticLambda17@43f0480
12-14 11:03:32.485 D/OpenGLRenderer(28611): endAllActiveAnimators on 0xb400007276b71910 (RippleDrawable) with handle 0xb400007106b60b00
12-14 11:03:32.490 E/a       (28611): Failed to create instance of class ig.a
12-14 11:03:32.490 E/a       (28611): java.lang.InstantiationException: java.lang.Class<ig.a> has no zero argument constructor
12-14 11:03:32.490 E/a       (28611): at java.lang.Class.newInstance(Native Method)
12-14 11:03:32.490 E/a       (28611): at ug.a.g(SourceFile:79)
12-14 11:03:32.490 E/a       (28611): at ug.a.a(SourceFile:1)
12-14 11:03:32.490 E/a       (28611): at g5.i.D(SourceFile:67)
12-14 11:03:32.490 E/a       (28611): at g5.i.l(SourceFile:19)
12-14 11:03:32.490 E/a       (28611): at q8.x.h(SourceFile:179)
12-14 11:03:32.490 E/a       (28611): at g5.c.H(SourceFile:561)
12-14 11:03:32.490 E/a       (28611): at androidx.emoji2.text.n.run(SourceFile:205)
12-14 11:03:32.490 E/a       (28611): at java.lang.Thread.run(Thread.java:1012)
12-14 11:03:32.495 W/ziparchive(  435): Unable to open '/data/app/~~s2Z1WOthYxY_54jYmqbA_A==/app.pachli.current-4lqmb6uNcrj9Vv_UD7UdJA==/split_config.arm64_v8a.dm': No such file or directory
12-14 11:03:32.495 W/ziparchive(  435): Unable to open '/data/app/~~s2Z1WOthYxY_54jYmqbA_A==/app.pachli.current-4lqmb6uNcrj9Vv_UD7UdJA==/split_config.arm64_v8a.dm': No such file or directory
12-14 11:03:32.496 W/.pachli.current(  435): Entry not found
12-14 11:03:32.497 W/ziparchive(  435): Unable to open '/data/app/~~s2Z1WOthYxY_54jYmqbA_A==/app.pachli.current-4lqmb6uNcrj9Vv_UD7UdJA==/split_config.xxhdpi.dm': No such file or directory
12-14 11:03:32.497 W/ziparchive(  435): Unable to open '/data/app/~~s2Z1WOthYxY_54jYmqbA_A==/app.pachli.current-4lqmb6uNcrj9Vv_UD7UdJA==/split_config.xxhdpi.dm': No such file or directory
12-14 11:03:32.498 W/.pachli.current(  435): Entry not found
12-14 11:03:32.500 D/nativeloader(  435): Configuring clns-4 for other apk /data/app/~~s2Z1WOthYxY_54jYmqbA_A==/app.pachli.current-4lqmb6uNcrj9Vv_UD7UdJA==/base.apk:/data/app/~~s2Z1WOthYxY_54jYmqbA_A==/app.pachli.current-4lqmb6uNcrj9Vv_UD7UdJA==/split_config.arm64_v8a.apk:/data/app/~~s2Z1WOthYxY_54jYmqbA_A==/app.pachli.current-4lqmb6uNcrj9Vv_UD7UdJA==/split_config.xxhdpi.apk. target_sdk_version=34, uses_libraries=, library_path=/data/app/~~s2Z1WOthYxY_54jYmqbA_A==/app.pachli.current-4lqmb6uNcrj9Vv_UD7UdJA==/lib/arm64:/data/app/~~s2Z1WOthYxY_54jYmqbA_A==/app.pachli.current-4lqmb6uNcrj9Vv_UD7UdJA==/base.apk!/lib/arm64-v8a:/data/app/~~s2Z1WOthYxY_54jYmqbA_A==/app.pachli.current-4lqmb6uNcrj9Vv_UD7UdJA==/split_config.arm64_v8a.apk!/lib/arm64-v8a:/data/app/~~s2Z1WOthYxY_54jYmqbA_A==/app.pachli.current-4lqmb6uNcrj9Vv_UD7UdJA==/split_config.xxhdpi.apk!/lib/arm64-v8a, permitted_path=/data:/mnt/expand:/data/user/0/app.pachli.current
12-14 11:03:32.514 V/GraphicsEnvironment(  435): Currently set values for:
12-14 11:03:32.514 V/GraphicsEnvironment(  435):   angle_gl_driver_selection_pkgs=[]
12-14 11:03:32.514 V/GraphicsEnvironment(  435):   angle_gl_driver_selection_values=[]
12-14 11:03:32.514 V/GraphicsEnvironment(  435): ANGLE GameManagerService for app.pachli.current: false
12-14 11:03:32.514 V/GraphicsEnvironment(  435): app.pachli.current is not listed in per-application setting
12-14 11:03:32.514 V/GraphicsEnvironment(  435): Neither updatable production driver nor prerelease driver is supported.
12-14 11:03:32.547 V/NativeCrypto(  435): Registering org/conscrypt/NativeCrypto's 288 native methods...
12-14 11:03:32.550 W/WindowOnBackDispatcher(28611): sendCancelIfRunning: isInProgress=falsecallback=android.view.ViewRootImpl$$ExternalSyntheticLambda17@ccfe8ac
12-14 11:03:32.579 D/vulkan  (  435): searching for layers in '/data/app/~~s2Z1WOthYxY_54jYmqbA_A==/app.pachli.current-4lqmb6uNcrj9Vv_UD7UdJA==/lib/arm64'
12-14 11:03:32.579 D/vulkan  (  435): searching for layers in '/data/app/~~s2Z1WOthYxY_54jYmqbA_A==/app.pachli.current-4lqmb6uNcrj9Vv_UD7UdJA==/base.apk!/lib/arm64-v8a'
12-14 11:03:32.580 D/vulkan  (  435): searching for layers in '/data/app/~~s2Z1WOthYxY_54jYmqbA_A==/app.pachli.current-4lqmb6uNcrj9Vv_UD7UdJA==/split_config.arm64_v8a.apk!/lib/arm64-v8a'
12-14 11:03:32.580 D/vulkan  (  435): searching for layers in '/data/app/~~s2Z1WOthYxY_54jYmqbA_A==/app.pachli.current-4lqmb6uNcrj9Vv_UD7UdJA==/split_config.xxhdpi.apk!/lib/arm64-v8a'
12-14 11:03:32.590 D/CompatibilityChangeReporter(  435): Compat change id reported: 160794467; UID 10406; state: ENABLED
12-14 11:03:32.618 D/AndroidRuntime(  435): Shutting down VM
12-14 11:03:32.618 E/AndroidRuntime(  435): FATAL EXCEPTION: main
12-14 11:03:32.618 E/AndroidRuntime(  435): Process: app.pachli.current, PID: 435
12-14 11:03:32.618 E/AndroidRuntime(  435): java.lang.RuntimeException: Unable to start activity ComponentInfo{app.pachli.current/app.pachli.SplashActivity}: java.lang.IllegalArgumentException: No enum constant z6.n.TRENDINGTAGS
12-14 11:03:32.618 E/AndroidRuntime(  435): at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3802)
12-14 11:03:32.618 E/AndroidRuntime(  435): at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3942)
12-14 11:03:32.618 E/AndroidRuntime(  435): at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:103)
12-14 11:03:32.618 E/AndroidRuntime(  435): at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:139)
12-14 11:03:32.618 E/AndroidRuntime(  435): at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:96)
12-14 11:03:32.618 E/AndroidRuntime(  435): at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2448)
12-14 11:03:32.618 E/AndroidRuntime(  435): at android.os.Handler.dispatchMessage(Handler.java:106)
12-14 11:03:32.618 E/AndroidRuntime(  435): at android.os.Looper.loopOnce(Looper.java:205)
12-14 11:03:32.618 E/AndroidRuntime(  435): at android.os.Looper.loop(Looper.java:294)
12-14 11:03:32.618 E/AndroidRuntime(  435): at android.app.ActivityThread.main(ActivityThread.java:8194)
12-14 11:03:32.618 E/AndroidRuntime(  435): at java.lang.reflect.Method.invoke(Native Method)
12-14 11:03:32.618 E/AndroidRuntime(  435): at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:552)
12-14 11:03:32.618 E/AndroidRuntime(  435): at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:971)
12-14 11:03:32.618 E/AndroidRuntime(  435): Caused by: java.lang.IllegalArgumentException: No enum constant z6.n.TRENDINGTAGS
12-14 11:03:32.618 E/AndroidRuntime(  435): at java.lang.Enum.valueOf(Enum.java:302)
12-14 11:03:32.618 E/AndroidRuntime(  435): at z6.n.valueOf(SourceFile:1)
12-14 11:03:32.618 E/AndroidRuntime(  435): at w6.j.a(SourceFile:105)
12-14 11:03:32.618 E/AndroidRuntime(  435): at u6.c.<init>(SourceFile:867)
12-14 11:03:32.618 E/AndroidRuntime(  435): at k5.f0.get(SourceFile:716)
12-14 11:03:32.618 E/AndroidRuntime(  435): at fd.a.get(SourceFile:14)
12-14 11:03:32.618 E/AndroidRuntime(  435): at f.n.a(SourceFile:149)
12-14 11:03:32.618 E/AndroidRuntime(  435): at androidx.activity.m.onCreate(SourceFile:30)
12-14 11:03:32.618 E/AndroidRuntime(  435): at androidx.fragment.app.c0.onCreate(SourceFile:1)
12-14 11:03:32.618 E/AndroidRuntime(  435): at app.pachli.SplashActivity.onCreate(SourceFile:21)
12-14 11:03:32.618 E/AndroidRuntime(  435): at android.app.Activity.performCreate(Activity.java:8621)
12-14 11:03:32.618 E/AndroidRuntime(  435): at android.app.Activity.performCreate(Activity.java:8599)
12-14 11:03:32.618 E/AndroidRuntime(  435): at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1456)
12-14 11:03:32.618 E/AndroidRuntime(  435): at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3784)
12-14 11:03:32.618 E/AndroidRuntime(  435): ... 12 more
12-14 11:03:32.618 E/a       (  435): ACRA caught a RuntimeException for app.pachli.current
12-14 11:03:32.618 E/a       (  435): java.lang.RuntimeException: Unable to start activity ComponentInfo{app.pachli.current/app.pachli.SplashActivity}: java.lang.IllegalArgumentException: No enum constant z6.n.TRENDINGTAGS
12-14 11:03:32.618 E/a       (  435): at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3802)
12-14 11:03:32.618 E/a       (  435): at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3942)
12-14 11:03:32.618 E/a       (  435): at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:103)
12-14 11:03:32.618 E/a       (  435): at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:139)
12-14 11:03:32.618 E/a       (  435): at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:96)
12-14 11:03:32.618 E/a       (  435): at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2448)
12-14 11:03:32.618 E/a       (  435): at android.os.Handler.dispatchMessage(Handler.java:106)
12-14 11:03:32.618 E/a       (  435): at android.os.Looper.loopOnce(Looper.java:205)
12-14 11:03:32.618 E/a       (  435): at android.os.Looper.loop(Looper.java:294)
12-14 11:03:32.618 E/a       (  435): at android.app.ActivityThread.main(ActivityThread.java:8194)
12-14 11:03:32.618 E/a       (  435): at java.lang.reflect.Method.invoke(Native Method)
12-14 11:03:32.618 E/a       (  435): at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:552)
12-14 11:03:32.618 E/a       (  435): at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:971)
12-14 11:03:32.618 E/a       (  435): Caused by: java.lang.IllegalArgumentException: No enum constant z6.n.TRENDINGTAGS
12-14 11:03:32.618 E/a       (  435): at java.lang.Enum.valueOf(Enum.java:302)
12-14 11:03:32.618 E/a       (  435): at z6.n.valueOf(SourceFile:1)
12-14 11:03:32.618 E/a       (  435): at w6.j.a(SourceFile:105)
12-14 11:03:32.618 E/a       (  435): at u6.c.<init>(SourceFile:867)
12-14 11:03:32.618 E/a       (  435): at k5.f0.get(SourceFile:716)
12-14 11:03:32.618 E/a       (  435): at fd.a.get(SourceFile:14)
12-14 11:03:32.618 E/a       (  435): at f.n.a(SourceFile:149)
12-14 11:03:32.618 E/a       (  435): at androidx.activity.m.onCreate(SourceFile:30)
12-14 11:03:32.618 E/a       (  435): at androidx.fragment.app.c0.onCreate(SourceFile:1)
12-14 11:03:32.618 E/a       (  435): at app.pachli.SplashActivity.onCreate(SourceFile:21)
12-14 11:03:32.618 E/a       (  435): at android.app.Activity.performCreate(Activity.java:8621)
12-14 11:03:32.618 E/a       (  435): at android.app.Activity.performCreate(Activity.java:8599)
12-14 11:03:32.618 E/a       (  435): at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1456)
12-14 11:03:32.618 E/a       (  435): at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3784)
12-14 11:03:32.618 E/a       (  435): ... 12 more

INSTALLATION_ID=7825a8bb-57da-439a-b892-be3136c5d303
DEVICE_FEATURES=android.hardware.sensor.proximity=true
  com.verizon.hardware.telephony.lte=true
  com.verizon.hardware.telephony.ehrpd=true
  android.hardware.telephony.ims.singlereg=true
  android.hardware.sensor.accelerometer=true
  android.software.controls=true
  android.hardware.faketouch=true
  android.software.telecom=true
  android.hardware.telephony.subscription=true
  com.google.android.feature.D2D_CABLE_MIGRATION_FEATURE=true
  android.hardware.telephony.euicc=true
  android.hardware.usb.accessory=true
  android.hardware.telephony.data=true
  android.hardware.sensor.dynamic.head_tracker=true
  android.software.backup=true
  android.hardware.touchscreen=true
  android.hardware.touchscreen.multitouch=true
  android.software.erofs=true
  android.software.print=true
  android.software.activities_on_secondary_displays=true
  android.hardware.wifi.rtt=true
  android.software.device_lock=true
  com.google.android.feature.PIXEL_2017_EXPERIENCE=true
  android.software.voice_recognizers=true
  android.software.picture_in_picture=true
  android.hardware.fingerprint=true
  android.hardware.sensor.gyroscope=true
  android.hardware.audio.low_latency=true
  android.software.vulkan.deqp.level=true
  android.software.cant_save_state=true
  com.google.android.feature.PIXEL_2018_EXPERIENCE=true
  android.hardware.security.model.compatible=true
  android.hardware.telephony.messaging=true
  com.google.android.feature.PIXEL_2019_EXPERIENCE=true
  android.hardware.telephony.calling=true
  android.hardware.opengles.aep=true
  android.hardware.bluetooth=true
  com.google.android.feature.PIXEL_2023_MIDYEAR_EXPERIENCE=true
  android.software.window_magnification=true
  android.hardware.telephony.radio.access=true
  android.hardware.camera.autofocus=true
  com.google.android.feature.GOOGLE_BUILD=true
  android.hardware.telephony.gsm=true
  android.hardware.telephony.ims=true
  android.software.incremental_delivery=true
  android.hardware.se.omapi.ese=true
  android.software.opengles.deqp.level=true
  vendor.android.hardware.camera.preview-dis.front=true
  com.google.android.feature.PIXEL_2022_MIDYEAR_EXPERIENCE=true
  android.hardware.camera.concurrent=true
  android.hardware.usb.host=true
  android.hardware.audio.output=true
  android.software.ipsec_tunnel_migration=true
  android.software.verified_boot=true
  android.hardware.camera.flash=true
  android.hardware.camera.front=true
  android.hardware.sensor.hifi_sensors=true
  android.hardware.se.omapi.uicc=true
  android.hardware.strongbox_keystore=true
  android.hardware.screen.portrait=true
  android.hardware.nfc=true
  com.google.android.feature.TURBO_PRELOAD=true
  com.nxp.mifare=true
  com.google.android.feature.PIXEL_2021_MIDYEAR_EXPERIENCE=true
  android.hardware.sensor.stepdetector=true
  android.software.home_screen=true
  android.hardware.context_hub=true
  vendor.android.hardware.camera.preview-dis.back=true
  android.hardware.microphone=true
  android.software.autofill=true
  android.software.securely_removes_users=true
  com.google.android.feature.PIXEL_EXPERIENCE=true
  android.hardware.bluetooth_le=true
  android.hardware.sensor.compass=true
  com.google.android.feature.GOOGLE_FI_BUNDLED=true
  android.hardware.touchscreen.multitouch.jazzhand=true
  android.hardware.sensor.barometer=true
  android.software.app_widgets=true
  com.google.android.feature.PIXEL_2020_MIDYEAR_EXPERIENCE=true
  android.hardware.telephony.carrierlock=true
  android.software.input_methods=true
  android.hardware.sensor.light=true
  android.hardware.vulkan.version=true
  android.software.companion_device_setup=true
  android.software.device_admin=true
  com.google.android.feature.WELLBEING=true
  android.hardware.wifi.passpoint=true
  android.hardware.camera=true
  android.software.credentials=true
  android.hardware.device_unique_attestation=true
  android.hardware.screen.landscape=true
  android.software.device_id_attestation=true
  com.google.android.feature.AER_OPTIMIZED=true
  android.hardware.ram.normal=true
  com.google.android.feature.NEXT_GENERATION_ASSISTANT=true
  com.google.android.feature.PIXEL_2019_MIDYEAR_EXPERIENCE=true
  android.software.managed_users=true
  android.software.webview=true
  android.hardware.sensor.stepcounter=true
  android.hardware.camera.capability.manual_post_processing=true
  android.hardware.camera.any=true
  android.hardware.camera.capability.raw=true
  android.hardware.vulkan.compute=true
  android.hardware.touchscreen.multitouch.distinct=true
  android.hardware.location.network=true
  android.software.cts=true
  android.hardware.camera.capability.manual_sensor=true
  android.software.app_enumeration=true
  com.google.android.apps.dialer.SUPPORTED=true
  android.hardware.camera.level.full=true
  com.google.android.feature.GMS_GAME_SERVICE=true
  android.software.game_service=true
  android.hardware.identity_credential=true
  android.hardware.wifi.direct=true
  android.software.live_wallpaper=true
  com.google.android.feature.GOOGLE_EXPERIENCE=true
  android.software.ipsec_tunnels=true
  com.google.android.feature.EXCHANGE_6_2=true
  com.google.android.feature.DREAMLINER=true
  android.hardware.audio.pro=true
  android.hardware.nfc.hcef=true
  android.hardware.location.gps=true
  com.google.android.feature.ADAPTIVE_CHARGING=true
  android.software.midi=true
  android.hardware.nfc.any=true
  android.hardware.nfc.ese=true
  android.hardware.nfc.hce=true
  android.hardware.hardware_keystore=true
  com.google.android.feature.PIXEL_2020_EXPERIENCE=true
  android.hardware.telephony.euicc.mep=true
  android.hardware.wifi=true
  android.hardware.location=true
  android.hardware.vulkan.level=true
  android.software.virtualization_framework=true
  com.google.android.feature.PIXEL_2021_EXPERIENCE=true
  android.hardware.keystore.app_attest_key=true
  com.google.android.feature.QUICK_TAP=true
  android.software.wallet_location_based_suggestions=true
  android.hardware.wifi.aware=true
  com.google.android.feature.PIXEL_2022_EXPERIENCE=true
  android.software.secure_lock_screen=true
  android.hardware.biometrics.face=true
  android.hardware.telephony=true
  android.software.file_based_encryption=true
  glEsVersion=3.2
ENVIRONMENT=getDataDirectory=/data
  getDataSystemDirectory=/data/system
  getDownloadCacheDirectory=/data/cache
  getExternalStorageDirectory=/storage/emulated/0
  getExternalStorageState=mounted
  getInternalMediaDirectories=[/system/media, /oem/media, /product/media]
  getLegacyExternalStorageDirectory=/sdcard
  getOdmDirectory=/odm
  getOemDirectory=/oem
  getProductDirectory=/product
  getProductServicesDirectory=/product_services
  getRootDirectory=/system
  getStorageDirectory=/storage
  getSystemExtDirectory=/system_ext
  getVendorDirectory=/vendor
  isExternalStorageEmulated=true
  isExternalStorageLegacy=false
  isExternalStorageManager=false
  isExternalStorageRemovable=false
SHARED_PREFERENCES=default.animateCustomEmojis=true
  default.confirmFavourites=false
  default.wellbeingModeLimitedNotifications=false
  default.language=handled_by_system
  default.hideTopToolbar=false
  default.animateGifAvatars=true
  default.defaultMediaSensitivity=false
  default.showSelfUsername=disambiguate
  default.useBlurhash=true
  default.schema_version=2023090201
  default.appTheme=black
  default.fontFamily=default
  default.absoluteTimeView=false
  default.enableSwipeForTabs=false
  default.fabHide=false
  default.showBotOverlay=true
  default.statusTextSize=medium
  default.uiTextScaleRatio=100
  default.showStatsInline=true
  default.updateNotificationFrequency=ALWAYS
  default.updateNotificationLastNotificationMs=1702542274884
  default.showCardsInTimelines=true
  default.confirmReblogs=true
  default.wellbeingHideStatsProfile=false
  default.showJankyAnimationWarning=false
  default.mainNavPosition=bottom
  default.wellbeingHideStatsPosts=false
  default.customTabs=false

--- a/core/database/src/main/kotlin/app/pachli/core/database/model/TabData.kt
+++ b/core/database/src/main/kotlin/app/pachli/core/database/model/TabData.kt
@@ -28,9 +28,9 @@ enum class TabKind(val repr: String) {
     LOCAL("Local"),
     FEDERATED("Federated"),
     DIRECT("Direct"),
-    TRENDING_TAGS("TrendingTags"),
-    TRENDING_LINKS("TrendingLinks"),
-    TRENDING_STATUSES("TrendingStatuses"),
+    TRENDING_TAGS("Trending_Tags"),
+    TRENDING_LINKS("Trending_Links"),
+    TRENDING_STATUSES("Trending_Statuses"),
     HASHTAG("Hashtag"),
     LIST("List"),
     BOOKMARKS("Bookmarks")

